### ISSUE TITLE
Hooke's Law video issue

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/website#309]
@ariel-phet noticed on 2 laptops that the homepage was freezing across multiple browsers on Windows 8, but most noticeably on Firefox.

I helped trace this down to the hookes-law.mp4, which playing that freezes both laptops.

It may be a GPU-specific issue or more of a Windows 8 issue, but we should try re-encoding the video and playing it on those laptops (coordinate with @ariel-phet).

I tested on Mac/Win7 and Win10 VM across browsers, and was not able to reproduce.

See https://github.com/phetsims/website/issues/303

**Comment by aaronsamuel137** [comment]
I assume you are talking about the homepage on phet-dev, since this has not been deployed to the main site yet.

I am not sure what to do with regard to re-encoding this video. I used ffmpeg to resize it, which is what I did with all of the other videos. I know that the videos provided to me by @phet-steele for the other sims didn't have this issue, so perhaps we could ask him what he did?

For now, I have put the un-resized video on phet-dev for testing. If that works, we'll know it was the way I resized it that caused the problem. If that doesn't work, we'll probably want to get a new video from @arouinfar since the original video may be bad somehow.

Assigning to @ariel-phet to test on phet-dev.
